cli/test/lib/cli_spec.js

```diff
@@ -474,15 +474,25 @@ describe('cli', () => {
       expect(run.start).to.be.calledWith({ ciBuildId: '123', group: 'staging' })
     })
 
-    it('call run with --auto-cancel-after-failures', () => {
+    it('calls run with --auto-cancel-after-failures', () => {
       this.exec('run --auto-cancel-after-failures 4')
       expect(run.start).to.be.calledWith({ autoCancelAfterFailures: '4' })
     })
 
-    it('call run with --auto-cancel-after-failures with false', () => {
+    it('calls run with --auto-cancel-after-failures with false', () => {
       this.exec('run --auto-cancel-after-failures false')
       expect(run.start).to.be.calledWith({ autoCancelAfterFailures: 'false' })
     })
+
+    it('calls run with --runner-ui', () => {
+      this.exec('run --runner-ui')
+      expect(run.start).to.be.calledWith({ runnerUi: true })
+    })
+
+    it('calls run with --no-runner-ui', () => {
+      this.exec('run --no-runner-ui')
+      expect(run.start).to.be.calledWith({ runnerUi: false })
+    })
```

Comment:
@mschile What happens when someone passes both CLI options? What do we do when people pass --headed and --headless together for example, do we error? (Maybe you have this case handled somewhere, I didn't look through the whole PR)

Reply:
Unfortunately, the `commander` module handles this slightly odd. If both options are provided, the last one wins. For `headed` and `headless`, it's handled differently since the `commander` module sees them are completely different and we throw our own error.